### DISCUSSION TITLE
[SANDBOX] Secret Key Authentication ne fonctionne pas avec une api key sandbox

### DISCUSSION THREAD

**Discussion by saissaoui** [PayGreen/documentation#28] [Q&A] [answered]
Bonjour,

J'ai voulu tester les appels à l'api en mode sandbox, j'ai commencé par la génération d'un token en appelant le endpoint :
`https://api.paygreen.fr/auth/authentication/sh_0XXXXXXXX/secret-key`
avec le header `"Authorization":"sk_XXXXX"` avec un id shop et un secret key générées via la sandbox, mais j'ai toujours des retours 401 : 

`{
    "message": "Invalid API key",
    "timestamp": "2024-01-29T09:39:04+00:00"
}`

Est ce qu'il y a des paramètres a ajouter pour la sandbox? 

Merci d'avance.

**Comment by AdrienBr** [comment, marked as answer]
Bonjour @saissaoui ,
Assurez-vous d'appeler le bon sous-domaine correspondant à notre sandbox : 

`https://sb-api.paygreen.fr`

Adrien

**Reply by saissaoui** [reply]
Merci @AdrienBr ça marche
